plugins/ocp/ocp-nvme.c

```diff
@@ -902,6 +902,7 @@ static int eol_plp_failure_mode(int argc, char **argv, struct command *cmd,
 /// Telemetry Log
 //global buffers
 static __le64 total_log_page_sz;
+__u8 *header_data;
```

Comment:
You should use static here to avoid leaking into the global namespace.

But I have to say these globals are just wrong IMO. The getter functions should return directly and not via globals. 

The fix itself is correct but I really don't like where this goes with these globals. But I don't maintain this so I don't care too much.

Reply:
I've updated it myself.